src/main/java/net/snowflake/ingest/utils/ParameterProvider.java

```diff
@@ -51,6 +54,9 @@ public class ParameterProvider {
   It reduces memory consumption compared to using Java Objects for buffering.*/
   public static final boolean ENABLE_PARQUET_INTERNAL_BUFFERING_DEFAULT = false;
 
+  public static final Constants.BdecParquetCompression BDEC_PARQUET_COMPRESSION_ALGORITHM_DEFAULT =
```

Comment:
I thought we want ZSTD to be the default?

Reply:
was the ZSTD already tested with all our tests?

Reply:
I have tested `GZIP, ZSTD, SNAPPY` previously, but I will run the tests again. The SDK has changed a lot since then.